public/app/plugins/datasource/tempo/datasource.ts

```diff
@@ -313,8 +313,16 @@ function serviceMapQuery(request: DataQueryRequest<TempoQuery>, datasourceUid: s
       const { nodes, edges } = mapPromMetricsToServiceMap(responses, request.range);
       nodes.fields[0].config = {
         links: [
-          makePromLink('Total requests', totalsMetric, datasourceUid),
-          makePromLink('Failed requests', failedMetric, datasourceUid),
+          makePromLink(
+            'Request rate',
+            `rate(${totalsMetric}{server="\${__data.fields.title}"}[$__interval])`,
+            datasourceUid
+          ),
+          makePromLink(
+            'Failed request rate',
+            `rate(${failedMetric}{server="\${__data.fields.title}"}[$__interval])`,
```

Comment:
I think it would make more sense to use `__data.fields.id`. At the moment title and id are the same thing but generally, title is meant to be human-readable so id should be more stable.